models/_sector/dex/trades/base/_schema.yml

```diff
@@ -766,3 +766,22 @@ models:
           seed_file: ref('gridex_base_base_trades_seed')
           filter:
             version: 1
+
+  - name: baso_finance_base_base_trades
+    meta:
+      blockchain: base
+      sector: dex
+      project: baso_finance
```

Comment:
Hi @sophie-analytics,
have you added `baso_finance` into `dex.info`?
https://github.com/duneanalytics/spellbook/blob/main/models/dex/dex_info.sql

Also please check for other similar dex.trades pr as well.

Reply:
@Hosuke DONE! I have added it now